### PR TITLE
(Wrapper)Update MCTWrapper documentation and increase default num_of_images fo…

### PR DESCRIPTION
**Pull request by KazunoriSumiya**
…r mixed precision

## Pull Request Description:

add comment to NOTE
change num_of_images 5 to 32

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: docs/api/api_docs/classes/Wrapper.html [hunk 1]
```diff
@@ -53,7 +53,12 @@ <h3>Navigation</h3>
 quantized model export, handling framework-specific configurations and
 Target Platform Capabilities (TPC) setup.</p>
 <p>Initialize MCTWrapper with default parameters.</p>
-<p>Users can update the following parameters in param_items:</p>
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>Users can update the following parameters in param_items.
```
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: It feels unnatural that the parameter explanations are so lengthy before the function description. Could we organize it as follows order?

...
The wrapper manages the complete quantization pipeline from model input to quantized model export, handling framework-specific configurations and Target Platform Capabilities (TPC) setup.
　↓
quantize_and_export(
　　　　　　float_model,
　　　　　　representative_dataset,
　　　　　　method='PTQ',
...
　↓
Examples
　↓
Initialize MCTWrapper with default parameters.
Users can update the following parameters in param_items:
PTQ
...

Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: This note is for low priority variable.
"Users can update the following parameters in param_items." should be not in note but main sentences.

File: docs/api/api_docs/classes/Wrapper.html [hunk 2]
```diff
@@ -68,35 +73,31 @@ <h3>Navigation</h3>
...
-<td><p>TPC version (use_internal_tpc=False)</p></td>
+<tr class="row-even"><td><p>sdsp_version</p></td>
+<td><p>‘3.14’</p></td>
+<td><p>SDSP version for TPC</p></td>
 </tr>
-<tr class="row-even"><td><p>activation_error_method</p></td>
+<tr class="row-odd"><td><p>activation_error_method</p></td>
 <td><p>mct.core.QuantizationErrorMethod.MSE</p></td>
-<td><p>Activation quantization error method</p></td>
+<td><p>Activation quantization error method (low priority)</p></td>
```
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: Please emphasize whether it is "low priority" or not so that it's immediately clear,
for example by making "low priority" bold or underlined.

Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: Can the "low priority" items be grouped toward the bottom of the table?
Since parameter input is unordered, they don't need to match the order of variable definitions in the code.

Could you please reorganize the variables in this order?  
* Common high-priority variables (e.g., SDSP version, save_model_path)  
* Variables for mixed precision  
* Variables for GPTQ  
* Low-priority variables(for XQuant)
* Low-priority variables(others(optimizer))

File: docs/api/api_docs/classes/Wrapper.html [hunk 3]
```diff
@@ -116,27 +117,47 @@ <h3>Navigation</h3>
...
+<tr class="row-even"><td><p>linear_collapsing</p></td>
+<td><p>True</p></td>
+<td><p>Enable linear layer collapsing (low priority)</p></td>
+</tr>
+<tr class="row-odd"><td><p>residual_collapsing</p></td>
+<td><p>True</p></td>
+<td><p>Enable residual connection collapsing (low priority)</p></td>
+</tr>
+<tr class="row-even"><td><p>distance_weighting_method</p></td>
+<td><p>See <a class="reference external" href="https://sonysemiconductorsolutions.github.io/mct-model-optimization/api/api_docs/classes/MixedPrecisionQuantizationConfig.html">MixedPrecisionQuantizationConfig</a></p></td>
```
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: I think it would be helpful to include the link to

https://sonysemiconductorsolutions.github.io/mct-model-optimization/api/api_docs/classes/MixedPrecisionQuantizationConfig.html#mpdistanceweighting

along with the section headings for clarity.

File: docs/api/api_docs/classes/Wrapper.html [hunk 4]
```diff
@@ -156,21 +177,37 @@ <h3>Navigation</h3>
...
+<td><p>Enable residual connection collapsing (low priority)</p></td>
 </tr>
 <tr class="row-even"><td><p>n_epochs</p></td>
 <td><p>5</p></td>
 <td><p>Number of training epochs for GPTQ</p></td>
 </tr>
 <tr class="row-odd"><td><p>optimizer</p></td>
-<td><p>None</p></td>
-<td><p>Optimizer for GPTQ training</p></td>
+<td><p>default of <a class="reference external" href="https://sonysemiconductorsolutions.github.io/mct-model-optimization/api/api_docs/methods/get_keras_gptq_config.html#model_compression_toolkit.gptq.get_keras_gptq_config">get_keras_gptq_config</a> or <a class="reference external" href="https://sonysemiconductorsolutions.github.io/mct-model-optimization/api/api_docs/methods/get_pytroch_gptq_config.html#model_compression_toolkit.gptq.get_pytorch_gptq_config">get_pytorch_gptq_config</a></p></td>
```
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html:275]: Does the link include an explanation of the optimizer's default values?
If not, it would be better to link to a section that does.

File: docs/api/api_docs/classes/Wrapper.html (hunk 3 above)
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: I would like to match the notation with "default of ~" below.
Would "default of MixedPrecisionQuantizationConfig" be appropriate?

File: docs/api/api_docs/classes/Wrapper.html [hunk 5]
```diff
@@ -116,27 +117,47 @@ <h3>Navigation</h3>
 </tr>
 </thead>
 <tbody>
-<tr class="row-even"><td><p>target_platform_version</p></td>
-<td><p>‘v1’</p></td>
-<td><p>Target platform version (use_internal_tpc=True)</p></td>
+<tr class="row-even"><td><p>sdsp_version</p></td>
+<td><p>‘3.14’</p></td>
+<td><p>SDSP version for TPC</p></td>
```
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: "SDSP version for TPC" alone doesn't clearly indicate its meaning.
Could you add an explanation about why TPC is specified using SDSP?
For example, https://github.com/SonySemiconductorSolutions/mct-model-optimization/pull/1639/changes/1223dc638020e63d2fa881672972e4407697b443

File: docs/api/api_docs/classes/Wrapper.html [hunk 6]
```diff
@@ -116,27 +117,47 @@ <h3>Navigation</h3>
...
 <td><p>False</p></td>
-<td><p>Use Hessian-based scores for mixed precision</p></td>
+<td><p>Use Hessian-based scores for mixed precision (low priority)</p></td>
 </tr>
-<tr class="row-even"><td><p>weights_compression_ratio</p></td>
-<td><p>None</p></td>
-<td><p>Weights compression ratio for resource util</p></td>
+<tr class="row-odd"><td><p>weights_compression_ratio</p></td>
+<td><p>0.75</p></td>
+<td><p>Weights compression ratio for resource util (0.0～1.0)</p></td>
```
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: Please add "for GPTQ" to all variables related to GPTQ, and "for mixed precision" to those related to mixed precision.
For example, weights compression ratio is for mixed precision.

File: docs/api/api_docs/classes/Wrapper.html (hunk 1 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: move "Users can update the following parameters in param_items." to main sentences

File: docs/api/api_docs/classes/Wrapper.html (hunk 2 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: making "low priority" bold

File: docs/api/api_docs/classes/Wrapper.html (hunk 6 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: add "for GPTQ" and "for mixed precision"

File: docs/api/api_docs/classes/Wrapper.html (hunk 3 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: change to   "default of MixedPrecisionQuantizationConfig"

File: docs/api/api_docs/classes/Wrapper.html (hunk 4 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html:275]: The documentation for get_keras_gptq_config() states: 
optimizer (OptimizerV2) – Keras optimizer to use for fine-tuning for auxiliary variables **with a default learning rate set to 0.2.**
However, the documentation for get_pytorch_gptq_config() states:
optimizer (Optimizer) – Pytorch optimizer to use for fine-tuning for auxiliary variables.
Shouldn't the documentation for get_pytorch_gptq_config() be revised?

File: docs/api/api_docs/classes/Wrapper.html (hunk 3 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: change link to  https://sonysemiconductorsolutions.github.io/mct-model-optimization/api/api_docs/classes/MixedPrecisionQuantizationConfig.html#mpdistanceweighting

File: docs/api/api_docs/classes/Wrapper.html (hunk 5 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: change SDSP comments

File: docs/api/api_docs/classes/Wrapper.html (hunk 2 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: change variables order

File: docs/api/api_docs/classes/Wrapper.html (hunk 1 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: change comments order

Comment (ueno-hiroshi002) [reply docs/api/api_docs/classes/Wrapper.html]: "Initialize MCTWrapper with default parameters." remain top block.
Please move it after Examples too.

...
The wrapper manages the complete quantization pipeline from model input to quantized model export, handling framework-specific configurations and Target Platform Capabilities (TPC) setup.
　↓
quantize_and_export(
　　　　　　float_model,
　　　　　　representative_dataset,
　　　　　　method='PTQ',
...
　↓
Examples
　↓
**Initialize MCTWrapper with default parameters.**
Users can update the following parameters in param_items:
PTQ
...

File: docs/api/api_docs/classes/Wrapper.html (hunk 5 above)
Comment (ueno-hiroshi002) [reply docs/api/api_docs/classes/Wrapper.html]: This is not tutorial, I think "Here, we use the settings for SDSP Converter 3.14." is not necessary. Please delete.

File: docs/api/api_docs/classes/Wrapper.html (hunk 4 above)
Comment (ueno-hiroshi002) [reply docs/api/api_docs/classes/Wrapper.html:275]: So, please revise get_xxxx_gptq_config documents.

File: model_compression_toolkit/wrapper/mct_wrapper.py
Comment (kawakami-masaki0) [inline model_compression_toolkit/wrapper/mct_wrapper.py]: - colon is unnecessary
`Import MCT:`
- 'set' is capital
`set framework, method, and other parameters`
`set parameters if needed`

File: model_compression_toolkit/wrapper/mct_wrapper.py [hunk 7]
```diff
@@ -41,82 +41,9 @@ class MCTWrapper:
     quantized model export, handling framework-specific configurations and
     Target Platform Capabilities (TPC) setup.
     """
-
```
Comment (kawakami-masaki0) [inline model_compression_toolkit/wrapper/mct_wrapper.py:44]: Please delete unnecessary change

File: docs/api/api_docs/classes/Wrapper.html [hunk 8]
```diff
@@ -53,7 +53,76 @@ <h3>Navigation</h3>
...
+<dd><p>Main function to perform model quantization and export.
+The wrapper manages the complete quantization pipeline from model input to quantized model export,
+handling framework-specific configurations and Target Platform Capabilities (TPC) setup.</p>
```
Comment (kawakami-masaki0) [inline docs/api/api_docs/classes/Wrapper.html]: The following sentence appears twice, but I think it's redundant.
`The wrapper manages the complete quantization pipeline from model input to quantized model export,
handling framework-specific configurations and Target Platform Capabilities (TPC) setup.`

File: model_compression_toolkit/wrapper/mct_wrapper.py (hunk 7 above)
Comment (KazunoriSumiya) [reply model_compression_toolkit/wrapper/mct_wrapper.py:44]: delete TAB

File: docs/api/api_docs/classes/Wrapper.html (hunk 8 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: delete this sentence

File: model_compression_toolkit/wrapper/mct_wrapper.py
Comment (KazunoriSumiya) [reply model_compression_toolkit/wrapper/mct_wrapper.py]: delete ":" and Change to uppercase

File: docs/api/api_docs/classes/Wrapper.html (hunk 5 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: delete "Here, we use the ・・・

File: docs/api/api_docs/classes/Wrapper.html (hunk 1 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: move "Initialize MCTWrapper with default parameters."

File: docs/api/api_docs/classes/Wrapper.html (hunk 4 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html:275]: revise get_xxxx_gptq_config documents.

File: docs/api/api_docs/classes/Wrapper.html (hunk 1 above)
Comment (ueno-hiroshi002) [reply docs/api/api_docs/classes/Wrapper.html]: "Initialize MCTWrapper with default parameters" is bold, but this is not appropriate for chapter title.
Revert the chapter title to "Parameters" and add "Initialize MCTWrapper with default parameters" below it.

File: docs/api/api_docs/classes/Wrapper.html [hunk 9]
```diff
@@ -52,8 +52,74 @@ <h3>Navigation</h3>
...
+</pre></div>
+</div>
+<p><strong>Initialize MCTWrapper with default parameters</strong></p>
+<p>Users can update the following parameters in param_items.</p>
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>The low priority variable can be left at its default value, so there is no need to specify it.
+Specify it as necessary, for example, if you receive a warning from the <a class="reference external" href="https://sonysemiconductorsolutions.github.io/mct-model-optimization/guidelines/XQuant_Extension_Tool.html">XQuant extension tool</a>.</p>
+</div>
 <p><strong>PTQ</strong></p>
```
Comment (ueno-hiroshi002) [inline docs/api/api_docs/classes/Wrapper.html]: The bold formatting on "PTQ", "PTQ, Mixed Precision", "GPTQ", and "GPTQ, Mixed Precision" makes it look like the chapter has moved from "Parameters" to the next chapter.
Please remove the bold from  "PTQ", "PTQ, Mixed Precision", "GPTQ", and "GPTQ, Mixed Precision".

File: model_compression_toolkit/wrapper/mct_wrapper.py [hunk 10]
```diff
@@ -600,6 +526,82 @@ def quantize_and_export(self, float_model: Any,
             ...     param_items=param_items
             ... )
 
+        **Initialize MCTWrapper with default parameters**
+
+        Users can update the following parameters in param_items.
+    
+        .. note::
+           The low priority variable can be left at its default value, so there is no need to specify it.
+           Specify it as necessary, for example, if you receive a warning from the `XQuant extension tool <https://sonysemiconductorsolutions.github.io/mct-model-optimization/guidelines/XQuant_Extension_Tool.html>`_.
```
Comment (ueno-hiroshi002) [inline model_compression_toolkit/wrapper/mct_wrapper.py]: XQuant extension tool
↓
XQuant Extension Tool

Comment (KazunoriSumiya) [reply model_compression_toolkit/wrapper/mct_wrapper.py]: change to "Extension Tool"

File: docs/api/api_docs/classes/Wrapper.html (hunk 1 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: Revert the chapter title to "Parameters" and add "Initialize MCTWrapper with default parameters" below it.

File: docs/api/api_docs/classes/Wrapper.html (hunk 9 above)
Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: remove the bold from "PTQ", "PTQ, Mixed Precision", "GPTQ", and "GPTQ, Mixed Precision".

File: docs/api/api_docs/classes/Wrapper.html (hunk 1 above)
Comment (ueno-hiroshi002) [reply docs/api/api_docs/classes/Wrapper.html]: Add linebreak between  "Parameters" and  "Initialize MCTWrapper with default parameters"

Comment (KazunoriSumiya) [reply docs/api/api_docs/classes/Wrapper.html]: Add linebreak

**Review by ueno-hiroshi002 — APPROVED** [review]
